onebusaway-android/google-services.json

```diff
@@ -1,186 +1,35 @@
 {
   "project_info": {
-    "project_number": "1083736323923",
-    "firebase_url": "https://api-project-1083736323923.firebaseio.com",
-    "project_id": "api-project-1083736323923",
-    "storage_bucket": "api-project-1083736323923.appspot.com"
+    "project_number": "893400419714",
+    "firebase_url": "https://oba-user-behavior.firebaseio.com",
+    "project_id": "oba-user-behavior",
+    "storage_bucket": "oba-user-behavior.appspot.com"
   },
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "1:1083736323923:android:75ea84a920f3dd26",
```

Comment:
I'm not sure why the YRT (York) profile was removed from google-services.json, but it needs to be added back in (Travis failed because of this).  If we download the latest google-services.json from Firebase console it should still contain YRT profile.

Reply:
Oh - this is still pointed at the dummy Firebase account - nevermind, we just need to point this back at the main OBA Firebase account before merging.

Reply:
@barbeau  yes. That's on my to do list.  We need to remove the dummy google-services.json  and keep the old one.